src/mumble/GlobalShortcutButtons.cpp

```diff
@@ -10,13 +10,29 @@
 
 #include "Global.h"
 
+#include <QRect>
 #include <QTimer>
 
 GlobalShortcutButtons::GlobalShortcutButtons(QWidget *parent) : QDialog(parent), m_ui(new Ui::GlobalShortcutButtons) {
 	m_ui->setupUi(this);
+	setModal(true);
 
 	connect(m_ui->addButton, &QPushButton::toggled, this, &GlobalShortcutButtons::toggleCapture);
 	connect(m_ui->removeButton, &QPushButton::clicked, this, &GlobalShortcutButtons::removeItem);
+
+	QWidget *searchDialog = parent;
+	while (searchDialog && !qobject_cast< QDialog * >(searchDialog)) {
+		searchDialog = searchDialog->parentWidget();
+	}
+	parentDialog = qobject_cast< QDialog * >(searchDialog);
```

Comment:
This doesn't handle the case in which no `QDialog` is found.

Reply:
In that case it would behave like it does now. But I will think about what can be done in that case. I guess I may be able to take the MainWindow into account :thinking:

Reply:
If we do not find a ``QDialog``, we now place it in the center of ``Global::get().mw``